src/components/PageToggle.tsx

```diff
@@ -33,13 +40,13 @@ export const PageToggle: React.FunctionComponent = () => {
 
   useEffect(() => {
     applySidebarStylesToIsland()
-  }, [$isNavOpen])
+  }, [$isNavOpen, applySidebarStylesToIsland])
```

Comment:
I think that if you're putting a function into the hook deps you need to wrap it in `useCallback` to prevent constant rerenders no?

Reply:
Ah fair, could also just move the function inside the hook instead, I don't foresee it ever being used outside of it.